rector.php

```diff
@@ -0,0 +1,117 @@
+        __DIR__ . '/packages/zend-json',
+        __DIR__ . '/packages/zend-layout',
+        __DIR__ . '/packages/zend-ldap',
+        __DIR__ . '/packages/zend-loader',
+        __DIR__ . '/packages/zend-locale',
+        __DIR__ . '/packages/zend-log',
+        __DIR__ . '/packages/zend-mail',
+        __DIR__ . '/packages/zend-markup',
+        __DIR__ . '/packages/zend-measure',
+        __DIR__ . '/packages/zend-memory',
+        __DIR__ . '/packages/zend-mime',
+        __DIR__ . '/packages/zend-mobile',
+        __DIR__ . '/packages/zend-navigation',
+        __DIR__ . '/packages/zend-oauth',
+        __DIR__ . '/packages/zend-openid',
+        __DIR__ . '/packages/zend-paginator',
+        __DIR__ . '/packages/zend-pdf',
+        __DIR__ . '/packages/zend-progressbar',
+        __DIR__ . '/packages/zend-queue',
+        __DIR__ . '/packages/zend-reflection',
+        __DIR__ . '/packages/zend-registry',
+        __DIR__ . '/packages/zend-rest',
+        __DIR__ . '/packages/zend-search',
+        __DIR__ . '/packages/zend-search-lucene',
+        __DIR__ . '/packages/zend-serializer',
+        __DIR__ . '/packages/zend-server',
+        __DIR__ . '/packages/zend-session',
+        __DIR__ . '/packages/zend-soap',
+        __DIR__ . '/packages/zend-stdlib',
+        __DIR__ . '/packages/zend-tag',
+        __DIR__ . '/packages/zend-text',
+        __DIR__ . '/packages/zend-timesync',
+        __DIR__ . '/packages/zend-tool',
+        __DIR__ . '/packages/zend-translate',
+        __DIR__ . '/packages/zend-uri',
+        __DIR__ . '/packages/zend-validate',
+        __DIR__ . '/packages/zend-version',
+        __DIR__ . '/packages/zend-view',
+        __DIR__ . '/packages/zend-wildfire',
+        __DIR__ . '/packages/zend-xml',
+        __DIR__ . '/packages/zend-xmlrpc',
+        __DIR__ . '/packages/zend-service',
+        __DIR__ . '/packages/zend-service-akismet',
+        __DIR__ . '/packages/zend-service-amazon',
+        __DIR__ . '/packages/zend-service-audioscrobbler',
+        __DIR__ . '/packages/zend-service-console',
+        __DIR__ . '/packages/zend-service-delicious',
+        __DIR__ . '/packages/zend-service-ebay',
+        __DIR__ . '/packages/zend-service-flickr',
+        __DIR__ . '/packages/zend-service-livedocx',
+        __DIR__ . '/packages/zend-service-rackspace',
+        __DIR__ . '/packages/zend-service-recaptcha',
+        __DIR__ . '/packages/zend-service-shorturl',
+        __DIR__ . '/packages/zend-service-slideshare',
+        __DIR__ . '/packages/zend-service-strikeiron',
+        __DIR__ . '/packages/zend-service-twitter',
+        __DIR__ . '/packages/zend-service-yahoo',
+
+        //__DIR__ . '/packages/zend-test',
+        //__DIR__ . '/packages/zend-service-windowsazure',
+    ]);
+    $parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_81);
+
+    // Define what rule sets will be applied
+    //$containerConfigurator->import(LevelSetList::UP_TO_PHP_53);
```

Comment:
remove dead code (commented out code)

Reply:
Removed file

Reply:
I meant commented out block of code. not the whole file.

Reply:
Composer change can't be committed, but the addon file needed to run can be still committed. but @falkenhawk has the final say here. especially since your instructions changed.

also, fill the rector version and PHP version you used to run. (composer commands to install?) so other developers can re-run things in the future.

Reply:
All rector stuff can be disregarded, it didn't work and I had to manually do the changes anyways.